wrappers/react/src/hotTableClass.tsx

```diff
@@ -220,39 +237,54 @@ class HotTableClass extends React.Component<HotTableProps, {}> {
    * @param {React.ReactElement} rendererElement React renderer component.
    * @returns {Handsontable.renderers.Base} The Handsontable rendering function.
    */
-  getRendererWrapper(rendererElement: React.ReactElement): typeof Handsontable.renderers.BaseRenderer | any {
+  getRendererWrapper(rendererElement: React.ReactElement): typeof Handsontable.renderers.BaseRenderer {
     const hotTableComponent = this;
 
     return function (instance, TD, row, col, prop, value, cellProperties) {
       const renderedCellCache = hotTableComponent.getRenderedCellCache();
+      const portalContainerCache = hotTableComponent.getPortalContainerCache()
+      const key = `${row}-${col}`;
+
+       // Handsontable.Core type is mising guid
+      const instanceGuid = (instance as unknown as { guid: string }).guid;
+
+      const portalContainerKey = `${instanceGuid}${key}`
```

Comment:
That's pretty nitpicky, but can we change the key to ``${instanceGuid}-${key}`` or something like that? Without that the keys are not too readable, for example, the keys for rows 0-2 and columns 0-1 look something like this:
```
ht_72b3657ec151e0730-0
ht_72b3657ec151e0730-1
ht_72b3657ec151e0731-0
ht_72b3657ec151e0731-1
ht_72b3657ec151e0732-0
ht_72b3657ec151e0732-1
```

I know that they're not meant to be read by the human eye, but it would make it easier for debugging.

Reply:
Agreed